src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -105,36 +159,51 @@ public ConnectionBuilder<TSourceType> Bidirectional()
             return this;
         }
 
+        /// <inheritdoc cref="FieldBuilder{TSourceType, TReturnType}.Name(string)"/>
         public ConnectionBuilder<TSourceType> Name(string name)
         {
             FieldType.Name = name;
             return this;
         }
 
+        /// <inheritdoc cref="FieldBuilder{TSourceType, TReturnType}.Description(string)"/>
         public ConnectionBuilder<TSourceType> Description(string description)
         {
             FieldType.Description = description;
             return this;
         }
 
+        /// <inheritdoc cref="FieldBuilder{TSourceType, TReturnType}.DeprecationReason(string)"/>
         public ConnectionBuilder<TSourceType> DeprecationReason(string deprecationReason)
         {
             FieldType.DeprecationReason = deprecationReason;
             return this;
         }
 
+        /// <summary>
+        /// Sets the default page size
```

Comment:
Why default?

Reply:
That's what it is.  If specified, the optional `first` or `last` field arguments sets the number of records to return, thereby becoming the page size for the request.

Reply:
```suggestion
        /// Sets the default page size.
```

Reply:
Maybe we should rename this member to `DefaultPageSize` for v4.0 ?

Reply:
If so, I might also make it accept a `int?` and remove the `ReturnAll` member.

Reply:
ok